components/brave_rewards/browser/buildflags/buildflags.gni

```diff
@@ -1,6 +1,5 @@
 import("//build/config/features.gni")
 
 declare_args() {
-  # anonize doesn't currently build on windows
-  brave_rewards_enabled = is_mac || is_linux
+  brave_rewards_enabled = is_mac || is_linux || is_win
```

Comment:
Do we need this one? Now we are enabled on all OS's

Reply:
It might be useful when we start to build android and also some people might want to build without it.  That being said it's also likely to break since we won't build it consistently that way. I think just leave it for now.

Reply:
sounds good